packages/shared/src/schema/judge/post.ts

```diff
@@ -1,5 +1,6 @@
 import { z } from 'zod';
+import { commonSchema } from './common';
 
 export const post = z.object({
-  inviteCode: z.string().uuid(),
+  ...commonSchema.shape,
```

Comment:
I don't see this in their docs anywhere; what was the rationale for not using `merge`?